python/sdk/test/integration_test.py

```diff
@@ -715,3 +715,43 @@ def test_standard_transformer_with_feast(integration_test_url, project_name, use
 
     assert resp.json()["instances"] == exp_resp["instances"]
     merlin.undeploy(v)
+
+@pytest.mark.integration
+def test_custom_model_without_artifact(integration_test_url, project_name, use_google_oauth):
```

Comment:
can we have test for custom model deployment with artifact?

Reply:
I plan to update it after finishing the sample docker model. The sample will contains webservice code, Dockerfile and Notebook. So I think I update it in the next PR

Reply:
Got it 👌🏻